tools/generate-models/run.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env zsh
```

Comment:
`zsh` is the default shell, if we have `bin/bash` in other places we should replace them too

Reply:
Oh, indeed it's a default one since Catalina. Let's keep it 👍. Other tools work fine with `bin/bash` so there's no need to update them now. Our Bitrise stacks are Catalina too.